internal/examples/supervisor/supervisor/supervisor.go

```diff
@@ -331,7 +329,7 @@ service:
 // 1) the remote config from OpAMP Server, 2) the own metrics config section,
 // 3) the local override config that is hard-coded in the Supervisor.
 func (s *Supervisor) composeEffectiveConfig(ctx context.Context, config *protobufs.AgentRemoteConfig) (configChanged bool, err error) {
-	var k = koanf.New(".")
+	k := koanf.New(".")
```

Comment:
Please remove unrelated changes here and elsewhere. If we want to reformat the source it can be a separate PR, ideally supported by a linter so that we consistent in all source code.

Reply:
sounds like a plan, removed https://github.com/open-telemetry/opamp-go/pull/352/commits/8f99ffa3a3ea13aa17e9825a8a2a24a23d5cec66